app/locales/es.js

```diff
@@ -0,0 +1,147 @@
+export default {
+  loading: '(Loading)',
+  classifier: {
+    back: 'Back',
+    backButtonWarning: 'Going back will clear your work for the current task.',
+    done: 'Done',
+    doneAndTalk: 'Done & Talk',
+    next: 'Next',
+    talk: 'Talk',
+    tutorialButton: 'Show the project tutorial',
+    miniCourseButton: 'Restart the project mini-course'
+  },
+  project: {
+    loading: 'Loading project',
+    disclaimer: 'This project has been built using the Zooniverse Project Builder but is not yet an official Zooniverse project. Queries and issues relating to this project directed at the Zooniverse Team may not receive any response.',
+    nav: {
+      about: 'About',
+      classify: 'Classify',
+      talk: 'Talk',
+      collections: 'Collect'
+    },
+    home: {
+      researcher: 'Words from the researcher',
+      about: 'About %(title)s',
+      metadata: {
+        statistics: '%(title)s Statistics',
+        classifications: 'Classifications',
+        volunteers: 'Volunteers',
+        completedSubjects: 'Completed Subjects',
+        subjects: 'Subjects'
+      },
+      talk: {
+        zero: 'Noone is talking about <strong>%(title)s</strong> right now.',
```

Comment:
Pretty sure there's a typo here - this should be no one instead of noone, right?

Reply:
'Noone' in UK English, I think. I guess it will ultimately get translated into `Nadie` for this file, but the English should be double checked for the project home page.

Reply:
Ah, sorry, wasn't aware of that difference between American and UK English! Thanks! Happy to make a quick PR on the English text, if we're in favor of it - but don't mean to imply the American 'no one' is correct and UK 'noone' is not :)